sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -48,59 +48,66 @@ export interface RetryOptions {
 }
 
 /**
- * Options to passed when creating a producer using the EventHubClient.
+ * The set of options that can be specified when creating an `EventHubProducer`.
+ * These can be specified when creating the producer via the `createProducer` method.
  */
 export interface EventHubProducerOptions {
   /**
    * @property
-   * The id of the partition to which the event should be sent. If no id is provided,
-   * the service will determine the partition to which the event will be sent.
+   * The identifier of the partition that the producer will be bound to.
+   * If an id is provided, all events sent using the producer will reach the same partition.
+   * If no id is provided, the service will determine the partition to which the event will be sent.
    */
   partitionId?: string;
   /**
    * @property
-   * Retry options for the send operation on the producer. If no value is provided here, the
-   * retry options set when creating the `EventHubClient` is used.
+   * The retry options used to govern retry attempts when an issue is encountered while sending.
+   * If no value is provided here, the retry options set when creating the `EventHubClient` is used.
    */
   retryOptions?: RetryOptions;
 }
 
 /**
- * Options that can be passed when sending a batch of events using the EventHubsClient.
+ * The set of options that can be specified to influence the way in which events are sent to the
+ * Event Hubs service.
  */
 export interface SendOptions {
   /**
    * @property
-   * If specified, Event Hubs will hash this string to map to a partitionId.
+   * The partition hashing key to associate with the event or batch of events.
    * It guarantees that messages with the same partitionKey end up in the same partition.
    * Specifying this will throw an error if the producer was created using a `paritionId`.
    */
   partitionKey?: string | null;
   /**
    * @property
-   * Can cancel the current operation.
+   * An implementation of `AbortSignalLike` to signal the request to cancel the operation.
```

Comment:
> An implementation of `AbortSignalLike` interface to signal the request to cancel the operation.

I feel the word "interface" helps here. Thoughts?

Also, should we add a line to hint at how to get an implementation? Maybe something like

> For example, use the the @azure/abort-controller to create an AbortSignal

Reply:
Sure, that's more accurate. I do like adding a line of where to get an implementation as well. :)